### PR TITLE
feat: document project deletion

### PR DESCRIPTION
**Pull request by pubudug**

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-5cjq04 Confirm Project Deletion in Admin Area](https://app.clickup.com/t/5cjq04)

**Comment by chewbeccy** [comment]
I'm still on it

File: docs/meshcloud.project.md [hunk 1]
```diff
@@ -76,4 +76,26 @@ In the **Project Users** section you can click the `-` button in the row of a us
...
+If you would like to delete a project which is no longer used, go to your **Account** settings. Navigate to **Projects** and click the button with the trash icon.
+
+When you want to delete a meshProject it is not completely removed in the cloud platforms and in the database, only flagged for deletion.
+
+The deletion procedure depends on the variaty of meshTenants under the project:
+1. a project contains exclusively public cloud meshTenats (AWS, Azure, GCP)
```
Comment (pubudug) [inline docs/meshcloud.project.md:84]: It's not only about public cloud tenants though. It's about the platforms for which we don't have resource existence check. Right now Kubernetes and Openshit also fall under this category. I think the title 1. should also be changed to capture those platforms.

Comment (chewbeccy) [reply docs/meshcloud.project.md:84]: You are right! I'll change this

**Pull request merged**